aptos-move/framework/aptos-framework/sources/configs/staking_config.spec.move

```diff
@@ -0,0 +1,6 @@
+spec aptos_framework::staking_config {
+    spec module {
+        use aptos_framework::timestamp;
+        invariant timestamp::is_operating() ==> exists<StakingConfig>(@aptos_framework);
```

Comment:
You'll need to update this after https://github.com/aptos-labs/aptos-core/pull/3557 right?

Reply:
Yes, you're right!